specifications/transactions/README.md

```diff
@@ -0,0 +1,61 @@
+
+However, re-use of subaddresses creates on-chain linkage and may constitute publicly identifiable material. To address this, participants can also leverage an off-chain API to produce a one-time use reference id for the on-chain payment.
+
+## Introduction
+
+### On-Chain
+
+A Diem transaction consists of a sender, a script, and a set of arguments. Currency transfer transactions contain a set of arguments including a recipient, an amount, a currency, and metadata. On success, the transaction would debit the sender's account by the specified amount in addition to gas (network usage) costs and credit the recipient's account. On failure, the transaction may entirely be discarded or committed to the blockchain in which case the sender would only be debited gas costs. Diem leverages the metadata to provide context behind a transaction. However, the metadata itself is not validated by the Diem blockchain or Move standard library.
+
+### Off-Chain
+
+Users do not have direct ability to send and receive payments on the Diem network and must do so through VASPs. One method to do so, is to store potentially personally identifiable information in a transaction metadata, Diem also supports an off-chain framework that can negotiate a one-time transaction identifier called a `reference_id`. The off-chain is also used by support travel rule compliance.
+
+### Refunds
+
+Diem has a notion of refund built directly into the framework when handling errant transactions, so that the refunding party can signal to the recipient the reason for the refund. User refunds, however, there is no strict requirement in how refunds are handled.
+
+## Common Flows
+
+### Transfer with Subaddresses
+
+Bob would like to send Alice 10 XUS:
+* Alice loads her VASP's mobile APP and generate a PaymentURI
+* The PaymentURI contains a subaddress that uniquely identifies Alice along with additional parameters specifying sending 10 XUS
+* Alice shares this with Bob via a QR code
+* The PaymentURI opens Bob's VASP's mobile app and confirms Bob's intent to send 10 XUS
+* When Bob confirms, his VASP debits his account 10 XUS and generates a Diem on-chain transaction that credits Alice's VASP
+* Within the transaction, Bob's VASP embeds a subaddress referencing Bob and the subaddress referencing Alice that was contained within the PaymentURI
+* Upon processing the transaction, Alice's VASP credits Alice's account 10 XUS
+
+### Transfer with Subaddress Exceeding Travel Rule Limits
+
+In the case that Bob would send Alice an amount that exceeds the travel rule limit, the flow changes:
+
+* Bob's VASP will initiate an off-chain KYC exchange with Alice's VASP
+* Upon successful completion of the exchange, Alice and Bob's VASP would have agreed on a `reference_id` and Alice's VASP would have provided a signature for the transaction metadata that Bob's VASP will submit to the chain
```

Comment:
I know this is still early days but we're going to need to link to or define here what that signing looks like, since its not just a signature over the metadata.

Reply:
I suppose you're trying to jump to reference ids here to suggest that they should be preferred?

Reply:
introduce some concepts and technical terms but leave that for the detailed spec... I know most specs don't have intros, but we don't have to be like most specs.